Libraries/PeriphDrivers/Include/MAX32690/wut.h

```diff
@@ -114,71 +114,77 @@ typedef void (*mxc_wut_complete_cb_t)(int result);
  * @brief      Initialize timer module clock.
  * @param      pres       Prescaler value. 
  */
-void MXC_WUT_Init(mxc_wut_pres_t pres);
+void MXC_WUT_Init(mxc_wut_regs_t *wut, mxc_wut_pres_t pres);
 
 /**
  * @brief      Shutdown timer module clock.
  */
-void MXC_WUT_Shutdown(void);
+void MXC_WUT_Shutdown(mxc_wut_regs_t *wut);
 
 /**
  * @brief      Enable the timer. 
  */
-void MXC_WUT_Enable(void);
+void MXC_WUT_Enable(mxc_wut_regs_t *wut);
 
 /**
  * @brief      Disable the timer.
  */
-void MXC_WUT_Disable(void);
+void MXC_WUT_Disable(mxc_wut_regs_t *wut);
 
 /**
  * @brief      Configure the timer.
  * @param      cfg  Pointer to timer configuration struct.
  */
-void MXC_WUT_Config(const mxc_wut_cfg_t *cfg);
+void MXC_WUT_Config(mxc_wut_regs_t *wut, const mxc_wut_cfg_t *cfg);
 
 /**
  * @brief   Get the timer compare count.
  * @return  Returns the current compare count.
  */
-uint32_t MXC_WUT_GetCompare(void);
+uint32_t MXC_WUT_GetCompare(mxc_wut_regs_t *wut);
 
 /**
  * @brief   Get the timer capture count.
  * @return  Returns the most recent capture count.
  */
-uint32_t MXC_WUT_GetCapture(void);
+uint32_t MXC_WUT_GetCapture(mxc_wut_regs_t *wut);
 
 /**
  * @brief   Get the timer count.
  * @return  Returns the current count.
  */
-uint32_t MXC_WUT_GetCount(void);
+uint32_t MXC_WUT_GetCount(mxc_wut_regs_t *wut);
 
 /**
  * @brief   Clear the timer interrupt.
  */
-void MXC_WUT_IntClear(void);
+__attribute__((deprecated("Use MXC_WUT_ClearFlags instead.  See wut.h for more details."))) void
+MXC_WUT_IntClear(void);
+
+/**
+ * @brief   Clear the timer interrupt.
+ */
+void MXC_WUT_ClearFlags(mxc_wut_regs_t *wut);
 
 /**
  * @brief   Get the timer interrupt status.
  * @return  Returns the interrupt status. 1 if interrupt has occurred.
  */
-uint32_t MXC_WUT_IntStatus(void);
+uint32_t MXC_WUT_IntStatus(mxc_wut_regs_t *wut);
```

Comment:
`MXC_WUT_IntStatus(...)` should be replaced with `MXC_WUT_GetFlags(...)` since that's using the old names.